### ISSUE TITLE
Failover playbook has a typo in role name.

### ISSUE THREAD

**Issue by tonykay** [samdoran/ansible-role-pgsql-replication#7]
In the README example of the failover playbook the role name uses a `-` not an `_`

 ```
- name: Get the current PostgreSQL Version
      import_role:
        name: samdoran.pgsql-replication
```
Solution:
re-write 3rd line as:

`    name: samdoran.pgsql_replication`

(Nice work BTW, worked flawlessly on a Tower 3.4.1 cluster)

**Comment by samdoran** [comment]
Thanks for reporting this. I recently renamed all my roles to changes dashes to underscores and I missed this example.

**Comment by samdoran** [comment]
Fixed by 1442558f315b19ea2922da1b364aee981e758fc0
